support-frontend/assets/pages/[countryGroupId]/student/helpers/buildCheckoutUrl.ts

```diff
@@ -34,5 +35,9 @@ export default function buildCheckoutUrl(
 	if (promoCode) {
 		urlSearchParams.set('promoCode', promoCode);
 	}
+	const { enableDigitalAccess } = getFeatureFlags();
+	if (enableDigitalAccess) {
+		urlSearchParams.set('enableDigitalAccess', 'true');
+	}
```

Comment:
If `enableDigitalAccess` is enabled, won't the checkout pick up the feature flag too? Just wondering why this needs to be threaded through as a query string arg. Sorry if I'm missing something!

Reply:
Yes your right, true is not required  just the switch name. I will create an update PR to do this.

Reply:
I think for testing purposes for Helene its good to route it through so she does not need to keep modifying the Url to add the featureFlag back in. Have done a PR to treat it as a switch not a parmUrl that might be useful?